backend/EasySpeak.Core/EasySpeak.Core.DAL/EasySpeak.Core.DAL.csproj

```diff
@@ -20,10 +20,6 @@
   <ItemGroup>
     <ProjectReference Include="..\EasySpeak.Core.Common\EasySpeak.Core.Common.csproj" />
   </ItemGroup>
-  
-  <ItemGroup>
-    <Folder Include="Migrations\" />
-  </ItemGroup>
 
```

Comment:
Are you sure?

Reply:
Actually, I didn`t remove it deleted itself, so I got it back)